### PR TITLE
Modifications for RH certification

### PR DESCRIPTION
**Pull request by srinivas-kandula**
Modified docker file with required metadata for RH certification
Added sample config file to bundle in image to support default configuration of collector in Openshift platform.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -54,8 +54,10 @@ endif
 #This rule need to be run on RHEL with podman installed.
 container_rhel: build
 	cp $(OUT_DIR)/$(ARCH)/$(BINARY_NAME) $(TEMP_DIR)
+	cp LICENSE $(TEMP_DIR)/license.txt
```
Comment (vikramraman) [inline Makefile]: @srinivas-kandula Just a note, don't use the master branch to build the container image for v1.0.3. Create a branch based on the released tag v1.0.3. 

I recently updated the Makefile version to 1.0.4 (for a future version). And the master branch also includes some recent changes (related to events) that are not yet released.

File: deploy/examples/collector.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
+    clusterName: openshift-demo-cluster
```
Comment (vikramraman) [inline deploy/examples/collector.yaml]: Does this differ from the existing example in anyway?

Comment (srinivas-kandula) [reply deploy/examples/collector.yaml]: Existing example has configMap metadata and proxy address that would not work in Openshift.

File: Makefile (hunk 1 above)
Comment (srinivas-kandula) [reply Makefile]: For RH certification will build collector from tag v1.0.3.

File: deploy/examples/collector.yaml (hunk 2 above)
Comment (vikramraman) [reply deploy/examples/collector.yaml]: Ah I see. Can you rename this file to `openshift-config.yaml` and add a comment at the top of the file stating this is a plain yaml file shipped by default in openshift containers.

Comment (srinivas-kandula) [reply deploy/examples/collector.yaml]: Done.

**Pull request merged**